hardhat.config.ts

```diff
@@ -61,9 +63,19 @@ const config: HardhatUserConfig = {
             url: process.env.NODE_URL || "",
             gasPrice: 3000000000,
         },
-        forking: {
+        arbitrum: {
+            chainId: chainIds.arbitrum,
             url: process.env.NODE_URL || "",
+            gasPrice: 100000000,
+            initialBaseFeePerGas: 100000000,
```

Comment:
Do these need to be explicitly set? I think it's better they get set via the flags id needed.